protected/controllers/AdminFileController.php

```diff
@@ -440,6 +440,9 @@ public function actionUpdate($id)
 
                 $this->redirect(array('view', 'id' => $model->id));
             }
+        } elseif (isset($_POST['delete_attr'])) {
+            $args = $_POST['FileAttributes'];
+            FileAttributes::model()->findByPk($args['id'])->delete();
         }
```

Comment:
@pli888 I tried to create a condition in `actionUpdate`, when I clicked the `Delete!` button, it threw this error:
```
Error 500
Trying to get property of non-object
```